### DISCUSSION TITLE
Premium Partner Conditions

### DISCUSSION THREAD

**Discussion by MattiSG** [OpenTermsArchive/terms-types#17] [Added]
Large marketplaces often end up creating a specific programme for identifying more “reliable”, “professional” or otherwise qualifying sellers.

Examples include:

### [Alibaba Gold Supplier](https://service.alibaba.com/page/knowledge?pageId=128&category=9207657&knowledge=24259336&language=en)

> Gold Supplier is a premium membership for suppliers on Alibaba.com. It provides an extensive number of promotional opportunities to maximize their exposure and return-on-investment. To qualify for a Gold Supplier membership, a supplier must complete an authentication and verification process by a reputable third-party security service provider appointed by Alibaba.com. Once approved, Gold Supplier members are authorized to display the icon to demonstrate their authenticity.

### [AirBnB Plus](https://www.airbnb.com/help/article/2195/airbnb-plus-program-terms-and-conditions)

> Airbnb Plus (“Plus” or “Program”) is a Program that allows Hosts to apply to receive a “Plus” classification that recognizes Listings with exceptional quality, comfort and style.

### [BlaBlaCar Super Driver](https://blog.blablacar.fr/about-us/super-driver)

> Le Programme permet aux conducteurs inscrits sur BlaBlaCar (les “Conducteurs”) qui se distinguent en satisfaisant certains critères spécifiques relatifs à leur compte BlaBlaCar et à leur trajets de covoiturages (les “Super Drivers”), d’obtenir un badge Super Driver.

**Comment by MattiSG** [comment]
## Recap table of provided examples

| URL                                                                                                          | Title                                         | Service   | Programme name | Requested or automatic? | Audience               |
|--------------------------------------------------------------------------------------------------------------|-----------------------------------------------|-----------|----------------|-------------------------|------------------------|
| [URL](https://service.alibaba.com/page/knowledge?pageId=128&category=9207657&knowledge=24259336&language=en) | ALIBABA.COM GOLD SUPPLIER™ SERVICES AGREEMENT | Alibaba   | Gold Supplier  | Upon application        | Sellers on marketplace |
| [URL](https://www.airbnb.com/help/article/2195/airbnb-plus-program-terms-and-conditions)                     | Airbnb Plus Program Terms and Conditions      | Airbnb    | Plus           | Upon application        | Hosts                  |
| [URL](https://blog.blablacar.fr/about-us/super-driver)                                                       | Conditions Générales Super Driver             | Blablacar | Super Driver   | Automatic               | Drivers and users      |

## Relevance

These documents describe how larger intermediation platforms offer a way for service or goods providers to distinguish themselves according to quality criteria they set forth.

## Tryptich

- Writer: `intermediation service`
- Audience: `providers of goods or services`
- Object: `eligibility criteria, limitations and benefits of a distinguished provider status programme`

## Naming

1. Distinguished Provider Conditions
2. Super Provider Conditions
3. Premium Provider Conditions
4. Distinguished Partner Conditions
5. Super Partner Conditions
6. Premium Partner Conditions

**Reply by MattiSG** [reply]
Based on a first round a feedback:

1. “provider” should be avoided in the name and audience to avoid ambiguity with the service providers themselves.
2. `writer` should always be a provider, not a service type.

So, updated proposals:

## Tryptich

- Writer: `intermediation service provider`
- Audience: `sellers of goods or services`
- Object: `eligibility criteria, limitations and benefits of a privileged seller status programme`

## Naming

#### Adjective

1. Premium
2. Distinguished
3. Super

#### Noun

1. Seller
2. Partner

#### Qualifier

1. Conditions

#### Based on given feedback, I would go with “Premium Partner Conditions”.

**Reply by lucielechardoy** [reply]
Yes ok Premium Partner Conditions sounds good to me too

**Comment by martinratinaud** [comment]
Thanks for this @MattiSG 

I see in the existing documentTypes.json those writers
- service provider
- paid-for goods or service provider
- search engine or intermediation service provider

It seems to me the writer here is the same that gives the `Terms of Service` or others and thus should be qualified as `service provider`

For the naming, I would go for `3. Premium Provider Conditions` as the audiense also refers at `providers`

**Reply by MattiSG** [reply]
Thanks @martinratinaud!
I agree with you, `intermediation service` is not a proper qualifier for the `writer`. I will use `intermediation service provider`, as used in the regulation ([article 2.3](https://eur-lex.europa.eu/eli/reg/2019/1150/oj)) 👍

**Comment by lucielechardoy** [comment]
Thanks @MattiSG 
I think that 'provider' has a closer meaning to 'business user' than 'partner', and I think that premium is a good broad category, so overall I would also vote for 3. Premium provider conditions

**Reply by MattiSG** [reply]
I am not sure all those programmes necessarily refer to a “business user” in the sense of the P2B regulation ([article 2.1](https://eur-lex.europa.eu/eli/reg/2019/1150/oj)):

> ‘business user’ means any private individual acting in a commercial or professional capacity who, or any legal person which, through online intermediation services offers goods or services to consumers for purposes relating to its trade, business, craft or profession

For example, Blablacar Super Driver is for drivers, who are not acting “for purposes relating to [their] trade, business, craft or profession”.

On the other hand, “provider” can be ambiguous with “service providers”, the term we use throughout Open Terms Archive to describe the actors that provide the online service we track documents of 🤔

**Comment by clementbiron** [comment]
`Premium Conditions` should not be enough ?

**Reply by MattiSG** [reply]
I believe this would be quite an ambiguous name: these conditions are only for actors offering goods or services on an intermediation platform. I would be concerned that `Premium Conditions` can be mixed up with, for example, a premium user program such as Amazon Prime.

**Comment by MattiSG** [comment]
Added via https://github.com/ambanum/OpenTermsArchive/pull/941!
